release.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+if [ ! -d ../gh-pages/.git ]; then
+    echo "Need to set up 'gh-pages' clone in directory above for this script to work"
+    echo "Run:"
+    echo "cd .."
+    echo "git clone git@github.com/falcosecurity/charts gh-pages"
+    echo "cd gh-pages"
+    echo "git checkout gh-pages"
+    exit 1
+fi
+
+helm package falco
+mv *.tgz ../gh-pages/
+pushd ../gh-pages
+git checkout gh-pages
+
+#TODO rename to falcosecurity before merge
+helm repo index . --url https://nibalizer.github.com/charts
```

Comment:
```suggestion
helm repo index . --url https://falcosecurity.github.com/charts
```
I have created the `gh-pages` branch here.